integrity-backend/input-dropbox-examples/generic-wavz-meta-content.json

```diff
@@ -0,0 +1,19 @@
+{
+    "contentMetadata": {
+        "dateCreated": "2022-04-14T01:27:46Z",
+        "extras": {
+            "localsignSoftware": "Webrecorder ArchiveWeb.page 0.7.8, using warcio.js 1.5.0",
+            "localsignPublicKey": "MHYwEAYHKoZIzj0CAQYFK4EEACIDYgAEoCCciTZZ2Km8TGGsl2s4c91CyIPEVFcxyU6LEbRgnQtiuRtugKGmyxLKpK41kPsiDAOvmxA74VMuFAhiaqE3FfdSAPdnM8+V2ugYY+uNHlQlVIakJnuNZx/AYj5AoknV",
+            "localsignSignature": "9AHneB8LTEP80DNXI997A5f4j2//rRW911t1scyjnxBoYJut3afw6quK1Tww+DdwcgVqlNjfVMtH4rUN1v2VuOgq5bidXpoulInYQR7oQAJe4Tlq8H9gt84pcb8oRP3I",
+            "waczVersion": "1.1.1",
+            "software": "Webrecorder ArchiveWeb.page 0.7.8, using warcio.js 1.5.0",
+            "dateCrawled": "2022-04-14T01:25:06.091Z",
+            "pages": {
+                "vngimrkldkpt682awhwrb": "http://example.com/"
+            },
+            "dropboxPath": "/mnt/store/dropbox/Dropbox/integrity-stg-starlinglab-org"
```

Comment:
Probably should figure out how to crop it to just /integrity-stg-starlinglab-org, maybe by passing in the "dropbox mount point"

Reply:
I suggest we put this in:
```
    "private":
    {
      "uploadFilename": "Benedict Lau - My Archive.wacz",
      "uploadDirectory": "/mnt/store/dropbox/Dropbox/integrity-stg-starlinglab-org"
    }
```

The file name could contain helpful info like a person's name. Also could be sensitive. Use full path as is, but decouple from Dropbox, we may receive this via HTTP.